### PR TITLE
Added configs and snippets for atom

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added make target to install atom plugins
* Added snippets
* Configured autocomplete paths
* Configured replace autocomplete paths for links
* Configured replace autocomplete paths for images
* Configured markdown writer 
* Configured markdown writer image uploader
* Configured markdown link creation

## Why
* So make documentation creation easy

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,6 +14,34 @@ export HTMLTEST_LOG_LEVEL ?= 2
 deps: packages/install/hugo \
 	  packages/install/htmltest
 
+# Install useful atom plugins
+deps/atom:
```
Comment (osterman) [inline Makefile]: add something like
```
@which apm >/dev/null || (echo "Install the atom editor"; exit 1)
```

File: Makefile [hunk 2]
```diff
@@ -14,6 +14,34 @@ export HTMLTEST_LOG_LEVEL ?= 2
...
+		language-markdown \
+		markdown-table-editor \
+		markdown-writer \
+		tool-bar-markdown-writer \
+		markdown-toc \
+		project-manager \
+		local-config \
+		autocomplete-paths
+
+	@echo "Please follow that steps on first run"
```
Comment (osterman) [inline Makefile]: `Now complete the setup by performing the following tasks:`

File: Makefile (hunk 1 above)
Comment (goruha) [reply Makefile]: Done

File: Makefile (hunk 2 above)
Comment (goruha) [reply Makefile]: Done

**Pull request merged**